core/script_debugger_remote.cpp

```diff
@@ -67,17 +67,20 @@ Error ScriptDebuggerRemote::connect_to_host(const String &p_host, uint16_t p_por
 
 	int port = p_port;
 
-	int tries = 3;
+	const int tries = 6;
```

Comment:
maybe make this something like
```c++
const int waits[] =  { 1, 10, 100, 1000, 1000 };
const int tries = sizeof(waits) / sizeof(int);
```
Or just have a 0 value at the end and we stop when we hit 0. Makes adding/removing tries a little less brittle maybe.

Reply:
Well I don't expect it to change often, so need need to make it more complex than it needs to be IMO.

Reply:
Values won't change again, I believe, as it's just 10^n for n = 0, 1, 2, 3.